corehq/messaging/smsbackends/apposit/views.py

```diff
@@ -0,0 +1,27 @@
+from corehq.apps.sms.api import incoming
+from corehq.apps.sms.views import IncomingBackendView
+from corehq.messaging.smsbackends.apposit.models import AppositBackend
+from django.http import HttpResponse, HttpResponseBadRequest
+
+
+class AppositIncomingView(IncomingBackendView):
+    urlname = 'apposit_incoming'
+
+    def get(self, request, api_key, *args, **kwargs):
+        return HttpResponseBadRequest("ERROR: Expected POST")
+
+    def post(self, request, api_key, *args, **kwargs):
+        fromAddress = request.POST.get('fromAddress')
+        toAddress = request.POST.get('toAddress')
```

Comment:
yep, leaving it there so we remember what param it is in case we have a need for it